harvester/harvest/ogm.py

```diff
@@ -328,7 +329,7 @@ def _get_commit_before_date(self, target_date: str) -> pygit2.Commit | None:
         from_timestamp = date_parser(target_date).timestamp()
         commits = [
             commit
-            for commit in local_repo.walk(local_repo.head.target, pygit2.GIT_SORT_TIME)
+            for commit in local_repo.walk(local_repo.head.target, SortMode.TIME)
```

Comment:
Interesting!  How did this present itself?  In the new version of `pygit2`, was `.GIT_SORT_TIME` not present anymore?

Reply:
I think `mypy` raised the error saying the `.GIT_SORT_TIME` was not available. 🤔